test/spec/style_layer.js

```diff
@@ -0,0 +1,90 @@
+describe('L.mapbox.styleLayer', function() {
+    var server;
+
+    beforeEach(function() {
+        server = sinon.fakeServer.create();
```

Comment:
You should install fake responses with known values for URLs that the tests in this file request, so that the tests don't break if we change the attributions for `streets-v8`, `hybrid-streets-v8`, etc.